flink-python/pyflink/fn_execution/coders.py

```diff
@@ -31,31 +32,31 @@
 FLINK_SCHEMA_CODER_URN = "flink:coder:schema:v1"
 
 
-__all__ = ['RowCoder', 'BigIntCoder', 'TinyIntCoder', 'BooleanCoder',
+__all__ = ['FlattenRowCoder', 'RowCoder', 'BigIntCoder', 'TinyIntCoder', 'BooleanCoder',
            'SmallIntCoder', 'IntCoder', 'FloatCoder', 'DoubleCoder',
            'BinaryCoder', 'CharCoder', 'DateCoder', 'TimeCoder',
            'TimestampCoder', 'ArrayCoder', 'MapCoder', 'DecimalCoder']
 
 
-class RowCoder(FastCoder):
+class FlattenRowCoder(FastCoder):
     """
-    Coder for Row.
+    Coder for Row. The result of decode will be list for the performance.
```

Comment:
The result of decode will be list for the performance. -> The decoded result will be flattened as a list of column values of a row instead of a row object.